OptimoveSDK/Sources/Classes/Optimobile.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  ViewController.swift
+//  Optimobile
+//
+//  Created by Barak Ben Hur on 04/04/2022.
+//
+
+import UIKit
+import OptimoveCore
+import CoreLocation
+
+class Optimobile {
+    
+    private static let internalQueue = DispatchQueue(label: "com.singletioninternal.queue",
+                                                     qos: .default,
+                                                     attributes: .concurrent)
+    private static let _sheard = Optimobile()
+    
+    public static var sheard: Optimobile {
+        get {
+            return internalQueue.sync {
+                _sheard
+            }
+        }
+    }
+    
+    public typealias deepLinkComplition = (_ deepLinkResolution: DeepLinkResolution) -> ()
+    public typealias inAppComplition = (_ deepLink: [AnyHashable : Any], _ message: NSDictionary?) -> ()
+    public typealias pushOpenedComplition = (_ action: String? , _ data: [AnyHashable : Any]?) -> ()
+    
+    public enum Abilities {
+        case deppLinking(responder: deepLinkComplition?),
+             inApp(responder: inAppComplition?, inAppConsentStrategy: InAppConsentStrategy),
+             pushOpened(responder: pushOpenedComplition?)
+    }
+    
+    private static var builder: KSConfigBuilder!
+    
+    private init() {}
+    
+    public static func configure(for tenantInfo: OptimoveTenantInfo, apiKey: String, secretKey: String, abilities: [Abilities]? = nil) {
+        Optimove.configure(for: tenantInfo)
+        builder = KSConfigBuilder(apiKey: apiKey, secretKey: secretKey)
```

Comment:
Maybe loop Kosta in on this question but i think we should look to hide any Kumulos / KS references, even if we just renamed the class to something like OptiMobile / OptimobileConfigBuilder

Reply:
Renaming passes to follow